www/include/reporting/dashboard/viewServicesLog.php

```diff
@@ -45,10 +45,8 @@
 /*
  *  Getting service to report
  */
-isset($_GET["host_id"]) ? $host_id = $_GET["host_id"] : $host_id = "NULL";
-isset($_POST["host_id"]) ? $host_id = $_POST["host_id"] : $host_id;
-isset($_GET["item"]) ? $service_id = $_GET["item"] : $service_id = "NULL";
-isset($_POST["item"]) ? $service_id = $_POST["item"] : $service_id;
+$host_id = filter_var($_GET['host_id'], FILTER_VALIDATE_INT) ?? filter_var($_POST['host_id'], FILTER_VALIDATE_INT);
+$service_id = filter_var($_GET['item'], FILTER_VALIDATE_INT) ?? filter_var($_POST['item'], FILTER_VALIDATE_INT);
```

Comment:
if `$_GET['item']` does not exist, there will be a PHP notice

Reply:
fix with https://github.com/centreon/centreon/pull/8419/commits/23185b9036383d869664a0590531e69abe7f06c2